build/buildpipeline/windows-es.groovy

```diff
@@ -9,6 +9,6 @@ simpleNode('Windows.10.Amd64.ClientRS4.ES.Open') {
     stage ('Build') {
         def logFolder = getLogFolder()
         def environment = "set ASPNETCORE_TEST_LOG_DIR=${WORKSPACE}\\${logFolder}"
-        bat "${environment}&.\\run.cmd -CI default-build"
+        bat "${environment}&.\\build.cmd -ci /p:SkipJavaClient=true"
```

Comment:
The jenkins agents don't have the JDK. We get Java coverage from VSTS PR validation.

Ideally, Jenkins would also run Java stuff, but I'm not willing to invest more time in Jenkins right now since we're focus on getting to VSTS.